cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -116,6 +116,10 @@ function _add_common_params() {
         params=" --enable-istio $params"
     fi
 
+     if [ $KUBEVIRT_PSA == "true" ]; then
```

Comment:
there is a little problem here when the variable doesn't exist
(also nit: one space too much)

/root/project/hyperconverged-cluster-operator/_kubevirtci/cluster-up//cluster/ephemeral-provider-common.sh: line 119: [: ==: unary operator expected

Reply:
Isn't it the globing issue? I think that if KUBEVIRT_PSA was quoted it wouldnt fail.

Reply:
`KUBEVIRT_PSA` isn't declared so it is not globing issue
bash finds `==` which is binary when the left side doesn't exist
KUBEVIRT_PSA should have a default false value in order to solve it
I will create later a PR for it

Reply:
https://github.com/kubevirt/kubevirtci/pull/935

Btw do we want to enable PSA as part of `cluster-provision/k8s/check-cluster-up.sh` ?
but this is other story

Reply:
>Btw do we want to enable PSA as part of cluster-provision/k8s/check-cluster-up.sh ?
but this is other story

We first need to fix testuite.